scss/tech.scss

```diff
@@ -1,42 +1,29 @@
 $tech-logo-size: 100px;
 
 .techs-container {
-  display: flex;
-  flex-wrap: wrap;
-  margin-left: 100px;
+  display: grid;
+  margin: 0 auto;
+  width: 900px;
```

Comment:
Instead of using this width, you can try modifying grid-template-columns to fixed value